index.html

```diff
@@ -74,15 +74,14 @@
 ```
 2. Create bandwidth limit rule:
 ```bash
-neutron qos-bandwidth-limit-rule-create \
-    --max-kbps 512 \
-    --max-burst 1024 \
-    example-policy
+neutron qos-bandwidth-limit-rule-create --max-kbps 512 \
+                                            --max-burst 1024 \
```

Comment:
I checked it also and when I do something like:
`[18:37:14] slawek@dell:~/Dokumenty/Openstack/mangelajo/qos_presentation_austin_2016$ git diff
diff --git a/index.html b/index.html
index 0f6e3cb..0d9c05e 100644
--- a/index.html
+++ b/index.html
@@ -74,9 +74,10 @@ neutron qos-policy-create example-policy
 ```
 2. Create bandwidth limit rule:
 ```bash
-neutron qos-bandwidth-limit-rule-create --max-kbps 512 \
-                                            --max-burst 1024 \
-                                            example-policy
+neutron qos-bandwidth-limit-rule-create \
+        --max-kbps 512 \
+        --max-burst 1024 \
+        example-policy
 ```
 3. Apply policy for port/network
 ```bash
`
then there is fine "one" indent there. Is it not fine for You?